eventstore/sql/create_events.sql

```diff
@@ -274,6 +275,11 @@ INSERT INTO events with
 		coalesce(vspace.name, space_guid::text) as space_name,
 		duration,
 		plan_guid,
+		(case
+			when resource_type = 'service'
+			then coalesce(vsp.unique_id, 'd5091c33-2f9d-4b15-82dc-4ad69717fc03')::uuid
+			else plan_guid
```

Comment:
This is the last place `plan_guid` is used, so we don't need to store it in this table (line 276/7) and can remove it from all queries of event data.

Reply:
I have pushed a pair of commits. This required more work than what you think, but actually allowed me to notice a existing bug.

The format of events is exposed into the forecast events endpoint, so paas-admin needs to pass the `unique_id`. I added two commits for this, one to get rid of the old `plan_guid` and other to make `forecast_events` support both

Reply:
> but actually allowed me to notice a existing bug.

I was wrong there was not a bug, the feature was working fine because in the https://github.com/alphagov/paas-billing/blob/9a1c2da2de72578ee1b88a266f5ba9377e2d28bc/eventstore/store_forecast_events.go#L49-L50 we were populating the column `plan_unique_guid` with the value of `ev.PlanGUID`

But that is now the change is more explicit.

Anyway, there is still some mixed dichotomy  all around.  
 - The input config.json call this `plan_guid` when it is actually the `plan_unique_id`
 - The resulting billable_events the call the property `PlanGUID, but it is actually the value from unique.  https://github.com/alphagov/paas-billing/blob/9fcbab753142c9c373babadc24e4b8a0bc264468/eventio/event_billable.go#L56-L57